weekly-mission(week2)/css/signin.css

```
@@ -1,158 +1,142 @@
-    background: var(--white);
-    width: 400px;
+.input-profile {
+  padding: 1.8rem 1.5rem;
+  border-radius: 0.8rem;
+  border: 0.1rem solid var(--gray-4);
+  background: var(--white);
 }
 
-input:focus {
-    border: 3px solid var(--primary);;
-    outline: none;
+.input-profile:focus {
+  border: 0.3rem solid var(--primary);
+  outline: none;
 }
 
-.input {
-    display: flex;
-    flex-direction: column;
-    gap: 24px;
+div.input {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2.4rem;
+  width: 100%;
+  font-size: 1.6rem;
 }
 
 .input-row {
-    display: flex;
-    flex-direction: column;
-    gap: 12px;
-    position: relative;
-}
-
-.input-row label {
-    color: var(--black, #000);
-    font-family: Pretendard;
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: normal;
-}
-
-button.login-btn {
-    display: flex;
-    padding: 16px 20px;
-    width: 400px;
-    justify-content: center;
-    align-items: center;
-    border-radius: 8px;
-    background: var(--gra-purpleblue-to-skyblue);
-    color: #f5f5f5;
-    white-space: nowrap;
-    text-decoration: none;
-    color: var(--Grey-Light, #F5F5F5);
-    font-family: Pretendard;
-    font-size: 18px;
-    font-style: normal;
-    font-weight: 600;
-    line-height: normal;
-    border: 1px solid var(--gray-4);
-    cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  position: relative;
+  width: 100%;
+}
+
+.input-label {
+  color: var(--black, #000);
+  font-size: 1.4rem;
+  line-height: normal;
+}
+
+.login-btn {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  padding: 1.6rem 2rem;
+  border-radius: 0.8rem;
+  border: 0.1rem solid var(--gray-4);
+  background: var(--gra-purpleblue-to-skyblue);
+  color: #f5f5f5;
+  color: var(--Grey-Light, #F5F5F5);
+  font-size: 1.8rem;
+  font-weight: 600;
+  line-height: normal;
 }
 
 footer {
-    border-radius: 8px;
-    border: 1px solid #CCD5E3;
-    background: #E7EFFB;
-    width: 400px;
-    padding: 12px 24px;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-}
-
-.footer-sns {
-    display: flex;
-    align-items: center;
-    gap: 16px;
-}
-
-footer span {
-    color: #373740;
-    font-family: Pretendard;
-    font-size: 14px;
-    font-style: normal;
-    font-weight: 400;
-    line-height: normal;
-}
-
-.eye-off {
-    width: 16px;
-    height: 16px;
-    position: absolute;
-    top: 48px;
-    right: 20px;
-    cursor: pointer;
-}
\ No newline at end of file
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 1.2rem 2.4rem;
+  border-radius: 0.8rem;
+  border: 0.1rem solid #CCD5E3;
+  background: #E7EFFB;
+}
+
+.footer-sns-login {
+  display: flex;
+  align-items: center;
+  gap: 1.6rem;
+}
+
+.footer-main {
+  color: #373740;
+  font-size: 1.4rem;
+  line-height: normal;
+}
+
+.eye {
```

사소하지만 아이콘은 요런 느낌도 좋을 것 같아요~

```suggestion
.eye-icon {
```